### ISSUE TITLE
Improve the landing page for https://papers.freebsd.org/

### ISSUE THREAD

**Issue by jkoshy** [freebsd/freebsd-papers#137]
The site's landing page currently displays a list of year numbers.

![image](https://github.com/freebsd/freebsd-papers/assets/326115/17c84c35-b03e-4e98-b29f-465b54419ba3)

This page would be more engaging if we could instead surface a (randomly selected) subset of the available presentations.

**Comment by ppaeps** [comment]
I agree.  Do we know anyone who understands how webpages are made? :)
